README.md

```diff
@@ -320,6 +320,35 @@ await service.initializePayment({
 });
 ```
 
+Also, [PayPal](https://support.bigcommerce.com/s/article/Connecting-with-PayPal) (also known as PayPal Commerce Platform) requires specific options to initialize the PayPal Smart Payment Button on checkout page that substitutes a standard submit button
+
+```html
+<!-- This is where the PayPal button will be inserted -->
+<div id="container"></div>
+```
+
+```js
+service.initializePayment({
+    methodId: 'paypalcommerce',
+    paypalcommerce: {
+        container: '#container',
+        submitForm: () => {
```

Comment:
Maybe explain what these callbacks are meant to do `submitForm`, `onValidate` and `onRenderButton`?